neomodel/match.py

```diff
@@ -125,6 +126,9 @@ def install_traversals(cls, node_set):
         traversal = Traversal(source=node_set, name=key, definition=rel.definition)
         setattr(node_set, key, traversal)
 
+re._alphanum_str = frozenset(
+    "_абвгдеёжзийклмнопрстуфхцчшщъыьэюяАБВГДЕЁЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯ\
+            abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ01234567890")
```

Comment:
I think theres an actual regex property for this

Reply:
It does not work with the Cyrillic alphabet. Now it works. Before Cyrillic symbols re.escape insert 2 backslashes and neomodel build incorrect query.

Reply:
https://docs.python.org/2/library/re.html#re.UNICODE we just need to set this flag where the regex is being used. Can you do that and put it in a separate MR from this one please. Thanks!